docs/reST/ref/draw.rst

```diff
@@ -100,6 +100,45 @@ object around the draw calls (see :func:`pygame.Surface.lock` and
 
    .. ## pygame.draw.rect ##
 
+.. function:: rects
+
+   | :sl:`draw a sequence of rectangles`
+   | :sg:`rects(surface, draw_sequence) -> None`
+
+   Draws a sequence of rectangles on the given surface.
+
+   :param surface: the surface to draw the rectangles on
+   :param draw_sequence: a sequence composed of (color, rect, width, radius) where:
+
+      - color: color to draw with, the alpha value is optional if using a tuple ``(RGB[A])``
+
+      - rect: the rectangle to draw, with its position and dimensions
+
+      - width (optional): used for line thickness or to indicate that the rectangle is to be
```

Comment:
missing periods for color, rect, and width

Reply:
Thanks